src/style.css

```
@@ -0,0 +1,149 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+a,
+body,
+h2,
+img {
+  margin: 0;
+}
+
+.product-card {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.product-card__imac {
+  width: 160px;
+  height: 134px;
+  margin-left: 3px;
+
+  margin-bottom: 40px;
+}
```

```suggestion
.product-card__img {
  width: 160px;
  height: 134px;
  display: block;
  margin: 0 auto 40px;
}
```

In product card can be another product `product-card__imac` bad names for this one